DacFxStronglyTypedModel/Utils.tt

```diff
@@ -1,4 +1,4 @@
-﻿//------------------------------------------------------------------------------
+﻿<#//------------------------------------------------------------------------------
```

Comment:
This was done because the utils.tt output file, util.cs, outputs no content and has no need for the copyright to be outputted. Because utils.tt is included in the other template files which already output the copyright, it was outputting the copyright twice in those output files. Adding <# keeps the copyright at the top of util.tt file and removes the duplication from the other files.